### PR TITLE
LateNight

### PR DESCRIPTION
**Pull request by synthor**
- made the crossfader looking more than a real mixer, without this sp…ecial background image
- duplicated the master vu meter to have it more in the view while mixxxin
  
  in 2ch mode it's shown between the channel meters
  in 4ch mode it's shown below the mastercontrol knobs

### CODE REVIEW

File: res/skins/LateNight/mixer.xml [hunk 1]
```diff
@@ -93,6 +93,78 @@
...
+															</Connection>
+														</VuMeter>
+													</Children>
+												</WidgetGroup>
+											</Children>
+											<Connection>
+												<ConfigKey>[Master],show_4decks</ConfigKey>
+												<BindProperty>visible</BindProperty>
+												<Transform>
+													<Invert/>
```
Comment (ywwg) [inline res/skins/LateNight/mixer.xml]: invert / not is redundant -- just don't transform it

Comment (ywwg) [inline res/skins/LateNight/mixer.xml]: or... just Not should be enough

**Comment by ywwg** [comment]
I know the formatting is already incredibly bad, but can you replace the tabs with spaces?

And, can you target this to 1.12 instead of master?

Lastly, have you signed our contributor agreement?

**Comment by synthor** [comment]
Heya, thanks for info. I'm new to all that stuff and already thought, that i messed up the tabs. On my machine they look good (4 spaces), but on github there were eight..

This is my first push to github ever. I wanted to target it to 1.12. I'll correct this tomorrow.

The transform invert not is from the wiki page. I'll also try if transform not will work.

Nope not signed yet. As mentioned above, that was my first try to contribute. :)

**Comment by ywwg** [comment]
Very good for a first contribution!  I'm very protective of the LateNight skin but I like the changes :)

**Comment by synthor** [comment]
Thanks. I only use two decks and didn't knew where to place it in four channel mode. So I just tried to put it left of the master knobs, which wasn't looking good. So I put it below them in line with the channel meters.

Have you seen the difference with the crossfader?

Looks like a real mixer now. :)

**Pull request closed without merging**